templates/zerver/register.html

```diff
@@ -102,7 +102,20 @@ <h1>{{ _('Create your account') }}</h1>
                     <input type='hidden' name='key' value='{{ key }}' />
                     <input type='hidden' name='timezone' id='timezone'/>
                     <label for="id_email" class="inline-block label-title">{{ _('Email') }}</label>
-                    <div id="id_email">{{ email }}</div>
+                    <div class="email-visibility-text">
+                        {% if EMAIL_ADDRESS_VISIBILITY == "1" %}
+                        All the admins, members and guests will be able to see this email address.
+                        {% elif EMAIL_ADDRESS_VISIBILITY == "3" %}
+                        Only admins will be able to see this email address.
+                        {% elif EMAIL_ADDRESS_VISIBILITY == "4" %}
```

Comment:
We generally prefer to avoid using these hardcoded values like "3", "4", you can instead pass constant values like `EMAIL_ADDRESS_VISIBILITY_EVERYONE: Realm.EMAIL_ADDRESS_VISIBILITY_EVERYONE` from backend and use that here.

Reply:
Okay.I will take care of that and will make the required changes through backend.